packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -195,6 +196,8 @@ export const Settings = ({
           <Redirect from="/clients" to="/settings#connected-services" noThrow />
           {/* NOTE: `/settings/avatar/change` is used to link directly to the avatar page within Sync preferences settings on Firefox browsers */}
           <Redirect from="/avatar/change" to="/settings/avatar/" noThrow />
+
+          <PageRecoveryPhoneRemove path="/recovery_phone/remove" />
```

Comment:
This should be feature-flagged (behind `enable2FABackupPhoneFeature`) and redirect back to `/settings` if feature is off, no? 
Also wondering if it would be prudent to add it to the content-server routes too

Reply:
Yea it should be. I think Laurens PR has all this connected up. If we didn't want to block on that I can do in follow up in https://github.com/mozilla/fxa/pull/18287.

Reply:
Flags are already available to use in `main`, and this route would use a different flag than the one Lauren used for the setup route (we're keeping flags separate for adding and using/removing phones). Non blocking exactly, but want to make sure we don't forget to feature flag before this goes to stage next week.